lms/templates/certificates/_accomplishment-banner.html

```diff
@@ -53,38 +44,26 @@ <h2 class="message-title hd-5 emphasized">${accomplishment_banner_opening}</h2>
-                    % if share_settings:
-                      <%
-                       facebook_share_text = _("I completed the {course_title} course on {platform_name}.").format(course_title=accomplishment_course_title, platform_name=platform_name)
-                       twitter_share_text = _("I completed a course on {platform_name}. Take a look at my certificate.").format(platform_name=platform_name)
-                       share_url = request.build_absolute_uri(reverse('certificates:html_view', kwargs=dict(user_id=str(user.id),course_id=unicode(course_id))))
-                       if share_settings.get('CERTIFICATE_FACEBOOK_TEXT', None):
-                            facebook_share_text = share_settings.get('CERTIFICATE_FACEBOOK_TEXT')
-                       if share_settings.get('CERTIFICATE_TWITTER_TEXT', None):
-                            twitter_share_text = share_settings.get('CERTIFICATE_TWITTER_TEXT')
-                      %>
-                      % if share_settings.get('CERTIFICATE_FACEBOOK', False):
-                        <button class="action action-share-facebook btn btn-overlay btn-small icon-only" id="action-share-facebook"
-                           onclick="FaceBook.share({
-                            share_text: '${facebook_share_text}',
-                            share_link: '${share_url}',
-                            picture_link: '${request.build_absolute_uri(course_image_url)}',
-                            description: '${_('Click the link to see my certificate.')}'
-                            });">
-                            <i class="icon fa fa-facebook-official" aria-hidden="true"></i>
-                            <span class="action-label">${_("Post on Facebook")}</span>
-                        </button>
-                      %endif
-                      % if share_settings.get('CERTIFICATE_TWITTER', False):
-                      <% twitter_url = 'https://twitter.com/intent/tweet?text=' + twitter_share_text + '&url='+ urllib.quote_plus(share_url)%>
-                        <button data-tooltip="${_('Share on Twitter')}"
-                          class="action action-share-twitter btn btn-overlay btn-small icon-only"
-                          title="${_('Share on Twitter')}"
-                          onclick="popupWindow('${twitter_url}', 'tweetWindow', 640, 480); return false;">
-                            <i class="icon fa fa-twitter" aria-hidden="true"></i>
-                            <span class="action-label">${_("Tweet this Accomplishment. Pop up window.")}</span>
-                        </button>
-                      %endif
+                    % if facebook_share_enabled:
+                      <button class="action action-share-facebook btn btn-overlay btn-small icon-only" id="action-share-facebook"
+                         onclick="FaceBook.share({
+                          share_text: '${facebook_share_text}',
```

Comment:
All this text needs to be escaped to be sure it's correct in an HTML and Javascript context like this.